app/views/admin/trips/_form.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= form_for [:admin, @trip] do |f| %>
+  <div class=''>
```

Comment:
I built this form as a series of divs. Is everyone ok with using bootstrap form-controls for styling?

Reply:
We'll probably want to go trough and look at styling as I don't think we have this throughout.
We can add a story for standardizing styling

Reply:
I've got a table for the forms right now, but we can merge this now and decide later.

Reply:
I'm cool with bootstrap form-controls for styling these forms.